middleware/csrf/csrf_test.go

```diff
@@ -233,3 +233,54 @@ func Test_CSRF_From_Cookie(t *testing.T) {
 	utils.AssertEqual(t, 200, ctx.Response.StatusCode())
 	utils.AssertEqual(t, "OK", string(ctx.Response.Body()))
 }
+
+func Test_CSRF_ErrorHandler_WithoutCookie(t *testing.T) {
+	app := fiber.New()
+
+	errHandler := func(ctx *fiber.Ctx, err error) error {
+		return ctx.Status(419).Send([]byte("without CSRF cookie"))
+	}
+
+	app.Use(New(Config{ErrorHandler: errHandler}))
+
+	app.Post("/", func(c *fiber.Ctx) error {
+		return c.SendStatus(fiber.StatusOK)
+	})
+
+	h := app.Handler()
+	ctx := &fasthttp.RequestCtx{}
+
+	// Without CSRF cookie
+	ctx.Request.Reset()
+	ctx.Response.Reset()
+	ctx.Request.Header.SetMethod("POST")
+	h(ctx)
+	utils.AssertEqual(t, 419, ctx.Response.StatusCode())
+	utils.AssertEqual(t, "without CSRF cookie", string(ctx.Response.Body()))
+}
+
+func Test_CSRF_ErrorHandler_InvalidCookie(t *testing.T) {
+	app := fiber.New()
+
+	errHandler := func(ctx *fiber.Ctx, err error) error {
+		return ctx.Status(419).Send([]byte("Empty/invalid CSRF token"))
+	}
+
+	app.Use(New(Config{ErrorHandler: errHandler}))
+
+	app.Post("/", func(c *fiber.Ctx) error {
+		return c.SendStatus(fiber.StatusOK)
+	})
+
+	h := app.Handler()
+	ctx := &fasthttp.RequestCtx{}
+
+	// Empty/invalid CSRF token
+	ctx.Request.Reset()
+	ctx.Response.Reset()
+	ctx.Request.Header.SetMethod("POST")
+	ctx.Request.Header.Set("X-CSRF-Token", "johndoe")
```

Comment:
This case does not set a invalid cookie. And `empty` also means `without cookie` right?

Reply:
> This case does not set a invalid cookie.

I fixed Test_CSRF_ErrorHandler_InvalidToken for comparison between generated uuid token and invalid token.

> And empty also means without cookie right?

Empty means that token is not exists in request.
And I add Test_CSRF_ErrorHandler_EmptyToken in 1ad5a618cb0da239c41a218d3b06029556e7e571.


Test_CSRF_ErrorHandler_InvalidToken for
https://github.com/gofiber/fiber/blob/19e6a4429d1f9b6ae5f818c4b66a7357c78e58e4/middleware/csrf/csrf.go#L88-L103

Test_CSRF_ErrorHandler_EmptyToken for
https://github.com/gofiber/fiber/blob/19e6a4429d1f9b6ae5f818c4b66a7357c78e58e4/middleware/csrf/csrf.go#L82-L86